ajax/libs/angular-ui-codemirror/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-ui-codemirror",
+  "filename": "ui-codemirror.min.js",
+  "version": "0.3.0",
+  "description": "This directive allows you to add CodeMirror to your textarea elements.",
+  "author": "https://github.com/angular-ui/ui-codemirror/contributors",
+  "license": "MIT",
+  "homepage": "https://angular-ui.github.com",
```

Comment:
@lcd78706 I think this is much better https://angular-ui.github.io/ui-codemirror/